dev-python/spyder-kernels/spyder-kernels-1.8.1-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Kernels used by spyder on its ipython console"
+HOMEPAGE="https://github.com/spyder-ide/spyder-kernels/
+	https://pypi.org/project/spyder-kernels/"
+SRC_URI="https://github.com/spyder-ide/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/cloudpickle[${PYTHON_USEDEP}]
+	>=dev-python/ipykernel-5.1.3[${PYTHON_USEDEP}]
+	>=dev-python/jupyter_client-5.3.4[${PYTHON_USEDEP}]
+	dev-python/pyzmq[${PYTHON_USEDEP}]
+	dev-python/wurlitzer[${PYTHON_USEDEP}]"
+
+DEPEND="test? (
+	dev-python/cython[${PYTHON_USEDEP}]
+	dev-python/flaky[${PYTHON_USEDEP}]
+	dev-python/matplotlib[${PYTHON_USEDEP}]
+	dev-python/mock[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/pandas[${PYTHON_USEDEP}]
+	sci-libs/scipy[${PYTHON_USEDEP}]
+	dev-python/xarray[${PYTHON_USEDEP}] )"
+
+distutils_enable_tests pytest
+
+python_test() {
+	if [[ ${EPYTHON} = python3.6 ]]; then
+		# this fails in py3.6 for some reason
+		sed -i -e 's:test_turtle_launch:_&:' \
+			spyder_kernels/console/tests/test_console_kernel.py || die
+	fi
+
+	pytest -vv || die "Tests fail with ${EPYTHON}"
```

Comment:
You don't really have to revbump to enable tests, you can just edit the old ebuild in place. That being said, I also get that weird pandas problem with this package, 
```
============================================ ERRORS ============================================
__________________ ERROR collecting spyder_kernels/utils/tests/test_nsview.py __________________
spyder_kernels/utils/tests/test_nsview.py:18: in <module>
    import pandas as pd
/usr/lib64/python3.6/site-packages/pandas/__init__.py:26: in <module>
    from pandas._libs import (hashtable as _hashtable,
/usr/lib64/python3.6/site-packages/pandas/_libs/__init__.py:4: in <module>
    from .tslibs import (
/usr/lib64/python3.6/site-packages/pandas/_libs/tslibs/__init__.py:4: in <module>
    from .conversion import normalize_date, localize_pydatetime, tz_convert_single
pandas/_libs/tslibs/conversion.pyx:234: in init pandas._libs.tslibs.conversion
    ???
E   AttributeError: type object 'pandas._libs.tslibs.conversion._TSObject' has no attribute '__reduce_cython__'
!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Everything else works.

Reply:
>  That being said, I also get that weird pandas problem with this package

This is interesting, I don't get any errors about pandas in spyder or spyder-kernels. This is the pandas I have installed:
```
[I] dev-python/pandas
Available versions:  0.24.2^t **9999*l^t {X doc full-support minimal test PYTHON_TARGETS="python3_6 python3_7"}
Installed versions:  0.24.2^t(19:57:38 29/01/20)(X -doc -full-support -minimal -test PYTHON_TARGETS="python3_6 python3_7")
Homepage:            https://pandas.pydata.org/
Description:         Powerful data structures for data analysis and statistics
```

What flags do you have enabled on pandas? I have this suspicion that maybe USE="X" is required on pandas for this to work.

Reply:
I tried with `X`, `-X`, `full-support`, `-full-support` without changes. It also looks like cython has something to do with it. 

I tried googling the error, but so far none of the workarounds worked for me. Well, they were mostly just "update to latest version"

Reply:
Hmm, strange, I just re-emerged pandas, and now tests in spyder stopped working for me. I get the same error you're getting.

Reply:
Update: `MAKEOPTS="-j1" emerge -1 pandas` fixes this for me? Does that work for you to?

Reply:
Weirdly... yes.

Reply:
> Weirdly... yes.

Packages like scipy, and pandas have always been kinda unstable when it comes to parallel building. I remember that when I installed pandas I had to use `MAKEOPTS="-j1"`to get it to install at all, same for scipy (https://bugs.gentoo.org/676640, https://bugs.gentoo.org/680036). I often have to do the same when updating these packages. 

In fact, I was extremely surprised today to find that pandas compiled *without* `MAKEOPTS="-j1"`, when I found that spyder's test broke, I was immediately suspicious about `MAKEOPTS="-j1"`. Anyway, this is an issue that should probably be fixed in the pandas ebuild, so I have reported it here: https://bugs.gentoo.org/710030

Reply:
Damn you're efficient. I also made a bug about it, 
https://bugs.gentoo.org/710062

Well they should definitely notice it now! :)